config/routes.rb

```diff
@@ -479,6 +484,11 @@ def patch(*)
 
       put "#{root_path}/second_factors_backup" => "users#create_second_factor_backup"
 
+      post "#{root_path}/create_passkey" => "users#create_passkey"
```

Comment:
I'm wondering if this needs to be a POST request since the controller action doesn't create any new resource.